powerquery-docs/HelperFunctions.md

```diff
@@ -171,7 +171,10 @@ Table.GenerateByPage = (getNextPage as function) as table =>
         // otherwise set the table type based on the columns of the first page
         if (firstRow = null) then
             Table.FromRows({})
-        else        
+		// check for empty first table
+		else if (Table.IsEmpty(firstRow[Column1])) then
+			firstRow[Column1]
+        else
```

Comment:
I wonder if we could remove the multiple copies of this code in TripPin and just refer to this page instead. Keeping them all in sync is kind of a pain.

Other versions of this code include:
- https://docs.microsoft.com/en-us/power-query/samples/trippin/5-paging/readme
- https://github.com/microsoft/DataConnectors/blob/master/samples/TripPin/8-Diagnostics/Table.GenerateByPage.pqm
- https://github.com/microsoft/DataConnectors/blob/master/samples/TripPin/9-TestConnection/Table.GenerateByPage.pqm

Reply:
The alternative would be to update them all.

Reply:
@ehrenMSFT , agreed--it would be really nice to remove the duplication!

Created #271 to address the first bullet you listed. 

Not sure how to best de-duplicate the latter to, as I think the idea is to provide runnable code from a simple Git clone. Maybe these should be manually synced? Or, would it be acceptable to replace them with notes instructing the user to download code from the appropriate URL?